input/parser.go

```diff
@@ -26,7 +26,7 @@ type LogHandler struct {
 	Raw    []byte
 	Result map[string]interface{}
 	Parser LogParser
-	Stats func(key string, delta int64)
+	Stats  func(key string, delta int64)
```

Comment:
Your Go formatting seems off by the way, I don't believe this is compliant with `go fmt`. I'll fix up on master.

Reply:
Actually, it looks OK.